src/screens/NoteNew/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react";
+import { API } from "aws-amplify";
+import { FormGroup, FormControl, ControlLabel } from 'react-bootstrap';
+import LoaderButton from '../../components/LoaderButton';
+import config from '../../config';
+import { s3Upload } from '../../libs/awsLib';
+import { NoteNewProps, NoteNewState } from './types';
+import './index.css'; 
+
+class NoteNew extends Component<NoteNewProps, NoteNewState> {
+  constructor(props) {
+    super(props);
+
+    this.file = null;
+    this.state = {
+      isLoading: null,
+      note: {
+        attachment: null,
+      },
+      content: "",
+      attachmentURL: null
+    };
+  }
+
+  createNote(note) {
+    return API.post("notes", "/notes", {
+      body: note
+    });
+  }
+
+  validateForm() {
+    const { content } = this.state;
+    return content.length > 0;
+  }
+
+  formatFilename(str) {
```

Comment:
Humm what's the difference:
```
formatFilename(str) {
...
}
```
and
```
formatFilename = (str) => {
...
}
```
Please investigate it.

Reply:
As I know, the first function is function declaration, the second is arrow function. In this case, both functions are available because we don't use `this` in their scope. But I will change this to arrow function (the second one) for consistent. Thanks!

Reply:
That's great. I mean we should follow consistency for project we're working.